bindings/aws/kinesis/kinesis.go

```diff
@@ -0,0 +1,344 @@
+package kinesis
```

Comment:
nit: please add license comments in each file.

Reply:
Added the license comments